### DISCUSSION TITLE
Support for MSSQL

### DISCUSSION THREAD

**Discussion by ghost** [tokern/piicatcher#191] [General]
I see that the support for MSSQL was removed 2 years ago. Any plans to bring it back?

**Comment by vrajat** [comment]
Hello,
There are no plans to support MSSQL because pymssql has been deprecated. Refer: https://github.com/tokern/piicatcher/commit/2ddd4c21f3f101ad1f0e9aefbee0b7a5dc6418a6

There are alternatives like pyodbc and turboodbc. Open to contributions to add support for MSSQL.
